lib/service/database/database_config.dart

```diff
@@ -0,0 +1,56 @@
+import 'package:flutter/material.dart';
+import 'package:logger/logger.dart';
+import 'package:path/path.dart';
+import 'package:path_provider/path_provider.dart';
+import 'package:phone_comparison_app/service/database/table/announcement_table.dart';
+import 'package:sqflite/sqflite.dart';
+import '../../../screen/announcement/model/announcement_model.dart';
+import 'database_exception.dart';
+
+class DatabaseConfig {
+  static final DatabaseConfig _singleton = DatabaseConfig._internal();
+  Database? _db;
+
+  DatabaseConfig._internal();
+
+  factory DatabaseConfig() {
+    return _singleton;
+  }
+
+  static const String _dbName = "silkronBulletin.db";
+  static const int _dbVersion = 1;
```

Comment:
@irfanSilkron explain kenapa pakai 1 instead of 2?

Reply:
to indicate yang this is the first release of the db. Kalau ada changes untuk db tu, baru tukar jadi version 2.